src/hotspot/share/memory/metaspaceShared.cpp

```diff
@@ -1815,6 +1818,14 @@ bool MetaspaceShared::use_full_module_graph() {
   return result;
 }
 
+void MetaspaceShared::set_disable_eager_init(const char* value) {
```

Comment:
strcasecmp is not platform dependent, why not use it for all? It does not need <string.h> included.

Reply:
According to https://en.wikipedia.org/wiki/C_string_handling, strcasecmp is for POSIX, BSD, stricmp is for Windows.